README.md

```diff
@@ -12,64 +12,165 @@ A resource may classify its messages into groupings,
-## Syntax
-
-As currently specified, a message resource looks like this (syntax highlighting only approximate):
+This work was [presented](https://www.youtube.com/watch?v=ksgm_B-uUCU)
+at the [2024 Unicode Tech Workshop](https://www.unicode.org/events/utw/2024/).
+
+### Why?
+
+The prior work on a new message format has identified the following challenges
+that go beyond or arise from defining the syntax and behavior of a single message,
+but which are not well addressed by existing resource formats:
+
+- The MessageFormat 2 message syntax is naturally multi-line due to its internal structure,
+  and multi-line values are not easy to use in many of the current resource formats.
+- Comments and metadata (set either by automation or manually)
+  are an important means for translators and developers to communicated with one-another,
+  but are irrelevant to the retrieval and display of the message.
+  Their attachment to messages needs to be well specified,
+  while being easy to read, write, and ignore.
+- Structured metadata needs a common schema in order to be universally understood.
+- Hierarchical groupings of messages need to be representable,
+  and metadata be assignable not only to individual messages,
+  but also groups of messages.
+- Many messages are composed of multiple localizable parts
+  (such as an HTML element with a localizable body and localizable attributes),
+  and it should be possible to express a compound message formed of multiple connected parts.
+  While MessageFormat 2 does support e.g. markup elements with option values,
+  this may make it difficult to segment a message's localizable parts from each other.
+- A purpose-built localization resource format should be well specified,
+  and designed from the ground up to work with multiple implementations.
+  Its design and capabilities need to account for existing localization formats,
+  and allow for the representation of messages and resources in those formats.
+
+Some of these aspects are well supported by existing formats,
+but no one resource format serves all of the identified use cases.
+For instance:
+
+- Comments in Gettext .po files come in multiple types and each [clearly attaches] to the following entry,
+  but the format's representation of multi-line values is rather clumsy.
+- Fluent supports composability via [attributes] and [message references],
+  but its FTL format is tightly coupled with its own message format.
+- XLIFF is an [OASIS Standard],
+  but it is primarily a machine interchange format that is not designed for human authoring
+  and its message representation does not provide for message variants.
+
+It would of course be possible to define a JSON or XML Schema
+for a resource format that would address all of the above issues.
+However, while JSON and XML are relatively easy to read, they are not easy to _write_.
+This would disadvantage in particular developers and translators
+who do not have access to tooling beyond a simple text editor.
```

Comment:
Applying suggestion made by @aphillips in https://github.com/eemeli/message-resource-wg/pull/24#discussion_r1925999984
```suggestion
This is a disadvantage to users, such as developers or translators,
who expect to use existing editors, especially simple text editors,
to create and manage these files.
```